src/functions-and-arrays.js

```diff
@@ -47,6 +171,25 @@ const wordsCount = [
   'matter'
 ];
 
+
+function howManyTimes (wordsCount, searchWord)  {
```

Comment:
I would add/replace an argument. The words array is not included as an argument right now. Plus, do we need the wordsCount as an argument?